ui-v2/app/controllers/dc/acls/index.js

```diff
@@ -27,6 +27,7 @@ export default Controller.extend(WithFiltering, {
       };
     });
   }),
+  // TODO: This should be using a searchable
   filter: function(item, { s = '', type = '' }) {
     const sLower = s.toLowerCase();
     return (
```

Comment:
Spotted I hadn't used the new searchables in the old acls system, actually I've just thought it will have been because it was old and I was changing as little as possible with the new ACL upgrade, possibly the 2nd PR to come in this series.